Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-relation.controller.ts

```diff
@@ -0,0 +1,186 @@
+                                                $scope.relationPaymentFrequenciesValue = result.response.availableFrequencyTypes[i];
+                                                break;
+                                            }
+                                        }
+                                        $scope.relationPaymentFrequenciesOptions = result.response.availableFrequencyTypes;
+
+                                    } else {
+                                        $scope.relationPaymentFrequenciesOptions = result.response.availableFrequencyTypes;
+                                    }
+
+                                    if ($scope.relationInterfacesValue) {
+
+                                        for (let i = 0; i < result.response.availableInterfaces.length; i++) {
+
+                                            if (result.response.availableInterfaces[i].id === $scope.relationInterfacesValue.id) {
+                                                $scope.relationInterfaceOptions = result.response.availableInterfaces;
+                                                $scope.relationInterfacesValue = result.response.availableInterfaces[i];
+                                                break;
+                                            }
+                                        }
+                                        $scope.relationInterfaceOptions = result.response.availableInterfaces;
+
+                                    } else {
+                                        $scope.relationInterfaceOptions = result.response.availableInterfaces;
+                                    }
+
+                                    if ($scope.relationContractsValue) {
+
+                                        for (let i = 0; i < result.response.availableContracts.length; i++) {
+
+                                            if (result.response.availableContracts[i].id === $scope.relationContractsValue.id) {
+                                                $scope.relationContractsOptions = result.response.availableContracts;
+                                                $scope.relationContractsValue = result.response.availableContracts[i];
+                                                break;
+                                            }
+                                        }
+                                        $scope.relationContractsOptions = result.response.availableContracts;
+
+                                    } else {
+                                        $scope.relationContractsOptions = result.response.availableContracts;
+                                    }
+
+                                }
+                                if ($scope.RelationExposedSystemData != null) {
+                                    $http
+                                        .get(`api/v1/systemrelations/options/${usageId}/in-relation-to/${$scope.RelationExposedSystemData.id}`)
```

Comment:
igen er det synd at der ingen service er, for så skal vi gentage de her kald igen og igen, men vi lader det ligge for nu